src/components/app/create/CreateApp.tsx

```diff
@@ -434,11 +441,21 @@ export class AddNewApp extends Component<AddNewAppProps, AddNewAppState> {
     renderFooterSection = (): JSX.Element => {
         return (
             <div className="w-800 dc__border-top flex right pt-16 pr-20 pb-16 pl-20 dc__position-fixed dc__bottom-0">
-                <button className="cta flex h-36" onClick={this.createApp}>
-                    {`${this.state.form.appCreationType === AppCreationType.Existing ? 'Clone ' : 'Create '}${
-                        this.props.isJobView ? 'Job' : 'App'
-                    }`}
-                </button>
+                <ButtonWithLoader
+                    rootClassName="flex cta h-36"
+                    onClick={this.createApp}
+                    disabled={this.state.createAppLoader}
+                    isLoading={false}
```

Comment:
pass `this.state.createAppLoader` to isLoading & remove custom progressing that's been added below added.